src/drivers/px4io/px4io.cpp

```diff
@@ -262,11 +264,11 @@ class PX4IO : public cdev::CDev
 	bool			_param_update_force;	///< force a parameter update
 
 	/* advertised topics */
-	orb_advert_t 		_to_input_rc;		///< rc inputs from io
-	orb_advert_t		_to_outputs;		///< mixed servo outputs topic
-	orb_advert_t		_to_servorail;		///< servorail status
-	orb_advert_t		_to_safety;		///< status of safety
-	orb_advert_t 		_to_mixer_status; 	///< mixer status flags
+	uORB::PublicationMulti<input_rc_s>			_to_input_rc{ORB_ID(input_rc)};				///< rc inputs from io
+	uORB::PublicationMulti<actuator_outputs_s>		_to_outputs{ORB_ID(actuator_outputs)};			///< mixed servo outputs topic
```

Comment:
I would remove all the comments. They are either redundant or wrong.

Reply:
Yes they're pretty useless. We should probably decide if we want to do doxygen for real or stop bothering.